Index: backend/.env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>MONGODB_URI=mongodb+srv://alishanali9703:YGWqnCZDHGjFdhhJ@clustert5.ukgsewn.mongodb.net/tutorial5?retryWrites=true\r\nJWT_SECRET=f8b7c4d9a2e3b4a6c7d8e9f0g1h2i3j4k5l6m7n8o9p0q1r2s3t4u5v6w7x8y9z0a1b2c3d4e5f6g7h8i9j0k1l2m3n4o5p6q7r8s9t0u1v2w3x4y5z6a7b8c9d0e1f2g3h4i5j6k7l8m9n0o1p2q3r4s5t6u7v8w9x0a1b2c3d4e5f6g7h8i9j0\r\nNODEMAIL_SERVICE=Gmail\r\nNODEMAIL_USERNAME=alishanali9703@gmail.com\r\nNODEMAIL_PASSWORD=gfio wptr mjmm eejt\r\nFROM_USERNAME=Flexi-Gym\r\nFROM_EMAILADDRESS=noreply@flexigym.com
===================================================================
diff --git a/backend/.env b/backend/.env
--- a/backend/.env	
+++ b/backend/.env	
@@ -1,4 +1,4 @@
-MONGODB_URI=mongodb+srv://alishanali9703:YGWqnCZDHGjFdhhJ@clustert5.ukgsewn.mongodb.net/tutorial5?retryWrites=true
+MONGODB_URI=mongodb+srv://root:root123@web.jorckuz.mongodb.net/?retryWrites=true&w=majority
 JWT_SECRET=f8b7c4d9a2e3b4a6c7d8e9f0g1h2i3j4k5l6m7n8o9p0q1r2s3t4u5v6w7x8y9z0a1b2c3d4e5f6g7h8i9j0k1l2m3n4o5p6q7r8s9t0u1v2w3x4y5z6a7b8c9d0e1f2g3h4i5j6k7l8m9n0o1p2q3r4s5t6u7v8w9x0a1b2c3d4e5f6g7h8i9j0
 NODEMAIL_SERVICE=Gmail
 NODEMAIL_USERNAME=alishanali9703@gmail.com
